Code/Modules/mutalisk/psp/pspTexture.cpp

```diff
@@ -51,18 +52,18 @@ void psp_texture::patchupTextureFromMemory(MtxHeader* header)
 	clut = (void*)(header->paletteOffset + (int)data);
 
 	swizzled = header->swizzle;
-//	printf("�� alloc = %x\n", (unsigned int)data);
+//	printf("¤¤ alloc = %x\n", (unsigned int)data);
 /*
-	printf("�� width = %i\n", width);
-	printf("�� height = %i\n", height);
-	printf("�� format = %i\n", format);
-	printf("�� stride = %i\n", stride);
-	printf("�� alloc = %x\n", (unsigned int)data);
-	printf("�� size = %i\n", header->vramAllocationSize);
-	printf("�� clut? = %i\n", clutFormat);
-	printf("�� clut# = %i\n", clutEntries);
-	printf("�� clut@ = %x\n", (unsigned int)clut);
-	printf("�� swizzled = %s\n", swizzled ? "yes" : "no");
+	printf("¤¤ width = %i\n", width);
+	printf("¤¤ height = %i\n", height);
+	printf("¤¤ format = %i\n", format);
+	printf("¤¤ stride = %i\n", stride);
+	printf("¤¤ alloc = %x\n", (unsigned int)data);
+	printf("¤¤ size = %i\n", header->vramAllocationSize);
+	printf("¤¤ clut? = %i\n", clutFormat);
+	printf("¤¤ clut# = %i\n", clutEntries);
+	printf("¤¤ clut@ = %x\n", (unsigned int)clut);
+	printf("¤¤ swizzled = %s\n", swizzled ? "yes" : "no");
```

Comment:
Was this symbol change intentional?
Depending on what that symbol represents, I feel like we should `#define SYMBOL "\xFOOBAR"` instead.

Reply:
no, i believe it came from git file encoding inconsistencies. I didnt even notice it was changed.